### PR TITLE
[FIXED-10392]: SQL Task Node Custom Parameters run failed

### LINKED ISSUE #10392: [Bug] [SQL] SQL Task Node Custom Parameters run failed
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

when i using Custom Parameters of SQL Task Node with OUT type, sql query result is more than one row, in downstream sql using the parameters failed, if upstream sql result is one row, it work fine.
failed work flow:
image

work flow success
image

### What you expected to happen

SQL Task Custom parameters work fine

### How to reproduce

add two sql task node, in first node query select and add custom parameter OUT type, in donwstream sql using the parameters 

### Anything else

_No response_

### Version

3.0.0-beta-1

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by liu-lewis**
## Purpose of the pull request

fix #10392

## Brief change log

modify: org.apache.dolphinscheduler.plugin.task.api.AbstractTaskExecutor#setSqlParamsMap return format SQL
add SqlTask test: org.apache.dolphinscheduler.plugin.task.sql.SqlTaskTest

### CODE REVIEW

**Comment by SbloodyS** [comment]
Hi @liu-lewis , please associate the issue.

**Comment by liu-lewis** [comment]
> Hi @liu-lewis , please associate the issue.

Ok, this is my first PR, i'm tring to do this better

**Comment by liu-lewis** [comment]
> Hi @liu-lewis , please associate the issue.

Hi @SbloodyS , this issue has closed by PR #11285 ,and this is my first PR, could you help me to check what improvements can I make for this PR so that I can submit it correctly next time.

**Comment by SbloodyS** [comment]
Please take a look when you have spare time. @zhuangchong

File: dolphinscheduler-task-plugin/dolphinscheduler-task-sql/src/test/resources/common.properties [hunk 1]
```diff
@@ -0,0 +1,108 @@
+#
```
Comment (caishunfeng) [inline dolphinscheduler-task-plugin/dolphinscheduler-task-sql/src/test/resources/common.properties:1]: Why create a common.properties here?

**Comment by davidzollo** [comment]
> > Hi @liu-lewis , please associate the issue.
> 
> Hi @SbloodyS , this issue has closed by PR #11285 ,and this is my first PR, could you help me to check what improvements can I make for this PR so that I can submit it correctly next time.

thanks for your first contribution,  does this mean that you solve the same issue with PR #11285?

**Pull request closed without merging**